### ISSUE TITLE
Screen reader announcement of numeric up-down and domain up-down controls reads as "Spinner spinner"

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#2370]
* .NET Core Version: (e.g. 3.0 Preview1, or daily build number, use `dotnet --info`)
* Have you experienced this same bug with .NET Framework?: Yes/No

**Problem description:**

**Actual behavior:** 
Accessible name and control type of numeric up-down and domain up-down controls are similar: "Spinner" and spinner.
Screen reader announces numeric up-down and domain up-down controls as "Spinner spinner"

**Expected behavior:**
Numeric up-down control should have "Numeric" accessible name and be announced as "Numeric spinner" (discussible, need to refer WPF or Win32).
Domain up-down control should have "Domain" accessible name and be announced as "Domain spinner" (discussible, need to refer WPF or Win32).

**Minimal repro:**
Create .NET Core WinForms app with numeric and/or domain up-down control/s.
Run Narrator and listen for control announcement.
Run Accessibility Insights and verify the accessibility of numeric and/or domain up-down control/s.

Please see https://github.com/microsoft/accessibility-insights-windows/issues/618 for details.
